docs/modules/rtctime.md

```diff
@@ -139,8 +139,10 @@ It is highly recommended that the timestamp is obtained via NTP (see [SNTP modul
 
 Values very close to the epoch are not supported. This is a side effect of keeping the memory requirements as low as possible. Considering that it's no longer 1970, this is not considered a problem.
 
+You specify the seconds parameter with the optional microseconds (defaults to 0). Setting the rate is independant.
```

Comment:
Generally speaking, docs don't use 2nd person like this.  Separately, that's remarkably terse.  How about "Times are specified across two arguments, seconds and microseconds.  If microseconds are not specified, the default is 0 (i.e., time is exactly at the boundary between two seconds).  In addition to the time, a third argument specifies the local clock's drift rate estimate, as documented in [`rtctime.get()`](#rtctimeget).  If this argument is not given or is `nil`, the current rate estimate will not be altered, and the rate estimate may be set without changing the time by leaving the seconds and microseconds arguments `nil`."

Reply:
Fixed.

As a side note -- this is the rate of a WEMOS D1 MINI over almost 24 hours:

![image](https://user-images.githubusercontent.com/1508813/89132156-10552600-d4e0-11ea-97c1-b6193d014ff0.png)

I suspect that differences are temperature related caused by localized heating due to the amount of work being done by the CPU. I'm going to try modifying a board to use a TCXO instead and see what happens.